rockcraft.yaml

```diff
@@ -8,16 +8,17 @@ base: ubuntu:22.04
 platforms:
   amd64:
 
-entrypoint: ["/bin/bash", "/opt/pebble-start.sh"]
+entrypoint: ["/bin/bash", "/opt/pebble/pebble-start.sh"]
 
 env:
   - SPARK_HOME: /opt/spark
+  - SPARK_CONFS: /etc/spark8t/conf
   - JAVA_HOME: /usr/lib/jvm/java-11-openjdk-amd64
-  - PYTHONPATH: /opt/spark/python:/opt/spark/python/dist:/usr/lib/python3/dist-packages
-  - PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/spark:/opt/spark/bin:/opt/spark/python/bin
+  - PYTHONPATH: /opt/spark/python:/opt/spark8t/python/dist:/usr/lib/python3/dist-packages
+  - PATH: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/spark:/opt/spark/bin:/opt/spark/python/bin:/opt/spark-client/python/bin
   - HOME: /home/spark
   - KUBECONFIG: /home/spark/.kube/config
-  - SPARK_USER_DATA: opt/spark/data
+  - SPARK_USER_DATA: var/lib/spark/data
```

Comment:
This was the requirement from the specification... We either need to adjust the snap (to current standards) -- or the specification in case we don't wanna keep this direction...

Reply:
See: https://docs.google.com/document/d/1MyT4Wt_-vgPwhwEjC-oeHRdLYrZWaQzFGTwclT7hD1U/edit#heading=h.gn80xntoximb

Reply:
I know I know, but in spark we don't quite have stateful data to be persisted (like Kafka) since sparkjobs are transients/ephemeral. This data is actually the folder where we store the `.scala_history`, and in the SNAP we set this to HOME. I believe we can do something similar here as well